hack/chains/README.md

```diff
@@ -1,57 +1,65 @@
-# hack/chains
+hack/chains
+===========
 
 Some bash scripts to help configure and demonstrate [Tekton
 Chains](https://github.com/tektoncd/chains) in an infra-deployments managed
 cluster.
 
-## To set up an environment from scratch
+To set up an environment from scratch
+-------------------------------------
 
 ### Bootstrap
 
-For CRC users:
+For CRC:
 
     crc delete
     crc start
     `crc console --credentials | tail -1 | cut -d\' -f2`
 
-Then (assuming you're in this branch on your own fork):
+Then (assuming you're working in a branch in your own fork of this repo):
 
     hack/bootstrap-cluster.sh preview
 
-A "go make some coffee" one liner:
+A "go make some coffee" one liner for CRC:
 
     cd $(git rev-parse --show-toplevel); crc delete; crc start; `crc console --credentials | tail -1 | cut -d\' -f2`; hack/bootstrap-cluster.sh preview
 
 Wait a while until you see mostly healthy/synced at [Argo CD](https://openshift-gitops-server-openshift-gitops.apps-crc.testing/applications).
 
-(I have degraded 'has' and 'spi' related to GitHub auth and I don't think it
-matters for the basic pipeline functionality needed to do builds and
-demonstrate chains.)
+If you didn't set up your GitHub org and credentials you might have degraded
+'has' and 'spi' apps, but I don't think it matters for the basic pipeline
+functionality needed to do builds and demonstrate chains.
+
+To get all the apps and components to go green, see [this
+guide](https://coreos.slack.com/files/T027F3GAJ/F036QJ81LLU).
 
 ### Create a key-pair signing secret for chains
 
     cd hack/chains
     ./create-signing-secret.sh
 
-### Apply some CA cert related hacks to make SSL work for the internal registry
+### Trust the cluster's SSL cert (optional)
 
-    ./trust-local-cert.sh
+Some of the demos expect that cosign running on your workstation can use SSL
+to connect to the cluster's internal registry. This script should will help
+make that work.
 
-## Demos
+    ./trust-local-cert.sh
 
-Note: All these demos have been tested in local CRC cluster. They haven't yet
-been confirmed working in a cluster-bot cluster.
+Demos
+-----
 
 ### Kaniko build demo
 
 - Trigger a taskrun that builds an image using kaniko and pushes it to
     the cluster's internal registry
 - Verify the image in the internal registry using cosign
 - Verify the image's attestation in the internal registry using cosign
-- Using rekor-cli verify the rekor record of the taskrun (and/or image?)
+- Use rekor-cli to verify the rekor record of the taskrun
 
-Because we're using the cluster registry, the trust-local-cert and
-setup-controller-certs are needed.
+Because we're using the cluster's internal registry, the demo will work better
+if your workstation trusts the cluster CA cert. This can be achieved by
+running `trust-local-cert.sh`.
```

Comment:
I'd like to explain what is needed for cluster-bot, but I'm not sure the answer...